cmd/kanikoExecute_test.go

```diff
@@ -332,39 +332,6 @@ func TestRunKanikoExecute(t *testing.T) {
 		cwd, _ := fileUtils.Getwd()
 		assert.Equal(t, []string{"--dockerfile", "Dockerfile", "--context", cwd, "--skip-tls-verify-pull", "--destination", "myImage:tag"}, execRunner.Calls[1].Params)
 	})
-	t.Run("success case - createBOM", func(t *testing.T) {
```

Comment:
why were the tests for kanikoExecute step being removed?

Reply:
It may not encapsulate syft, but it's still decreasing coverage. There should be something at least to check accessibility and input parameters.